content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one will try to solve?
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as 
+there are:
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
+- Enabling of free speech and freedom of software
```

Comment:
I would remove this last bullet if it were me, "free speech" and "freedom of software" are vaguely defined topics that people love to have hand wavy arguments about.

Also in the US right now given the recent bans of hate speech websites being in the news adding this bullet may read as a coded message for those people to come to XMPP since right wing extremist groups often co-opt the language of the left and use "free speech" to mean "I want to spout hate speech on the internet and everyone bans me for it", and we don't want to be sending that message unintentionally. Of course, when this is out of the news this isn't an association I'd normally make, but at this time I'd argue we shouldn't make it sound like we're advocating for their so-called "free speech" platforms (which have nothing to do with the actual concept of free speech).

Reply:
I think switch to `Enabling more sovereignty of communication` is better, at least I want to make a statement that you gain back control in this direction (for the good or the bad)